.github/workflows/ci.yml

```diff
@@ -0,0 +1,79 @@
+name: "CI"
+
+env:
+  DOTENV_CONFIG_PATH: "./.env.example"
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - main
```

Comment:
This is different across our repos. I think we probably want to run on every push ( or push on open pulls) and on pull requests for main and dev.
Using push and pull request like this will cause duplicate jobs to run. We should prevent that so we don't clog the queue